src/pytest_netdut/factories.py

```diff
@@ -17,6 +17,7 @@
 import logging
 import re
 import pytest
+from packaging import version
```

Comment:
Could we use the parser also? https://packaging.pypa.io/en/latest/version.html#packaging.version.parse

Reply:
Yeah, using a standard implementation of version parsing would be better. Was there an issue with it?